python/arcticdb/version_store/_normalization.py

```diff
@@ -39,14 +37,8 @@
 
 PICKLE_PROTOCOL = 4
 
-try:  # 0.21+ Compatibility
-    from pandas._libs.tslib import Timestamp
-    from pandas._libs.tslibs.timezones import get_timezone
-except ImportError:
-    try:  # 0.20.x Compatibility
-        from pandas._libs.tslib import Timestamp, get_timezone
-    except ImportError:  # <= 0.19 Compatibility
-        from pandas.tslib import Timestamp, get_timezone
+from pandas._libs.tslib import Timestamp
```

Comment:
I suspect this is the same as `pd.Timestamp`?

Reply:
I should think so